scripts/docker_cmake

```diff
@@ -24,7 +24,7 @@ ARGS=(
     -D TPL_ENABLE_Libmesh=ON
         -D Libmesh_INCLUDE_DIRS=$LIBMESH_DIR/include
         -D Libmesh_LIBRARY_DIRS=$LIBMESH_DIR/lib
-    -D TPL_ENABLE_MOAB=ON
+    -D TPL_ENABLE_MOAB=${ENABLE_MOAB}
```

Comment:
I can try but I just want to point out that if `ENABLE_MOAB` is empty, moab will be turned on. So the behavior of `docker_cmake` hasn't changed.